CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 13.2.2 - 2019-02-20
+### Changed
+* Added a retry mechanism to handle AWS S3 copying bug
+* Added a configuration parameter to support a configurable number of maximum attempts at copying
```

Comment:
this line can go, enough we just mention the retry I think. Can you add a link to the issue see examples below

Reply:
Or make it 
```Added a configurable retry mechanism to handle flaky AWS S3 to S3 copying```.

Reply:
Thanks both, will make this more concise